app/src/components/features/rules/RulePairs/Pairs/Rows/RowsMarkup/ResponseBodyRow/index.js

```diff
@@ -106,7 +109,64 @@ const ResponseBodyRow = ({ rowIndex, pair, pairIndex, ruleDetails, isInputDisabl
               >
                 {pair.response.value ? "Change file" : " Select file"}
               </RQButton>
-              <span className="file-path"> {pair.response.value ? pair.response.value : " No file chosen"}</span>{" "}
+              {pair.source.operator === GLOBAL_CONSTANTS.RULE_OPERATORS.MATCHES ||
```

Comment:
why also for matches?

Reply:
We can capture groups from both regex and wildcard matches